libgnucash/backend/xml/sixtp-utils.cpp

```diff
@@ -138,58 +141,61 @@ concatenate_child_result_chars (GSList* data_from_children)
  */
 
 
+template <typename T>
+static bool parse_chars_into_num (const char* str, T* num_ptr)
+{
+    if (!str)
+        return false;
+
+    while (std::isspace (*str))
+        ++str;
+
+    const char* end_ptr = str + std::strlen (str);
+
+    auto res = std::from_chars (str, end_ptr, *num_ptr);
+    if (res.ec != std::errc{})
+        return false;
+
+    while (std::isspace (*res.ptr))
+        ++res.ptr;
+
+    return (res.ptr == end_ptr);
+}
+
 /*********/
 /* double
  */
 
 gboolean
 string_to_double (const char* str, double* result)
 {
-    char* endptr = 0x0;
-
-    g_return_val_if_fail (str, FALSE);
-    g_return_val_if_fail (result, FALSE);
+#if __cpp_lib_to_chars >= 201611L
```

Comment:
It may be worth adding a comment here to explain this is due to clang not having std::from_char yet for doubles ? That may help to remember it's (hopefully) a temporary workaround.

Reply:
Done...